flytekit/core/promise.py

```diff
@@ -684,6 +690,7 @@ def binding_data_from_python_std(
     t_value: Any,
     t_value_type: type,
     nodes: List[Node],
+    is_union_type_variants_expanded: bool = True,
```

Comment:
can we remove this now with the other change?

Reply:
Done